scripts/mbedtls_framework/code_wrapper/psa_wrapper.py

```diff
@@ -0,0 +1,283 @@
+"""Generate wrapper functions for PSA function calls.
+"""
+
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+import argparse
+import itertools
+import os
+from typing import Any, Iterator, List, Dict, Collection, Optional, Tuple
+
+from .. import build_tree
+from .. import c_parsing_helper
+from .. import c_wrapper_generator
+from .. import typing_util
+
+from .psa_buffer import BufferParameter
+
+class PSAWrapperCFG:
+    input_headers = ['crypto.h', 'crypto_extra.h']
```

Comment:
Because the class was originally a dictionary and was considered [bad implementation](https://github.com/Mbed-TLS/mbedtls-framework/pull/25#discussion_r1654996924) in the previous round of reviews. 

And given that is a dataclass there is no need to consider instance variables. When we override, the members tha are outdates should be overriden.

Reply:
> given that is a dataclass there is no need to consider instance variables.

I can't figure out why something being a “dataclass” has anything to do with considering instance variables. I'm not even sure what you call a dataclass. This isn't what [Python itself calls a dataclass](https://docs.python.org/3/library/dataclasses.html).

Reply:
This is not addressed as of 03bfc58000f7e6ff35a0f4a97821619c054ee275.

Reply:
Can you elaborate on how that should be addressed?

Reply:
(continuing at https://github.com/Mbed-TLS/mbedtls-framework/pull/25#discussion_r1682463049)